### PR TITLE
LEAF-3198 auto loading classes

### PR DESCRIPTION
**Pull request by jampaul3**
Removed all includes and requires for classes.
Added one require for the LEAF_autoloader.

### CODE REVIEW

**Review by shaneodd — CHANGES_REQUESTED** [review]

File: LEAF_Nexus/scripts/convertFileNames.php [hunk 1]
```diff
@@ -1,106 +0,0 @@
-<?php
```
Comment (raphaelfretter) [inline LEAF_Nexus/scripts/convertFileNames.php:1]: Is this file no longer needed?

File: LEAF_Request_Portal/db_mysql.php [hunk 2]
```diff
@@ -1,347 +0,0 @@
-<?php
```
Comment (raphaelfretter) [inline LEAF_Request_Portal/db_mysql.php:1]: Not sure about this either.

**Review by raphaelfretter — COMMENTED** [review]
It's finally done. Just a couple questions about some files.

File: LEAF_Request_Portal/db_mysql.php (hunk 2 above)
Comment (jampaul3) [reply LEAF_Request_Portal/db_mysql.php:1]: This one should be deleted. When I was going through every file, and noticing an issue with what db file was being used and then noticing there really wasn't much of a difference between the two files (Nexus and Portal) I decided to combine them. That made the two obsolete so yes I will delete this one.

File: LEAF_Nexus/scripts/convertFileNames.php (hunk 1 above)
Comment (jampaul3) [reply LEAF_Nexus/scripts/convertFileNames.php:1]: You can double check but when I search for the file name it does not come up. So best as I can tell it is not used anywhere.

**Review by raphaelfretter — APPROVED** [review]

**Review by shaneodd — APPROVED** [review]

**Pull request merged**